R/tm_data_table.R

```diff
@@ -362,21 +355,3 @@ srv_data_table <- function(id,
     })
   })
 }
-
```

Comment:
removing get_datanames_selected() causes teal.gallery::launch_app("exploratory") and other apps to break. Should it be moved to utils.R / zzz.R?